drivers/FlashIAP.h

```diff
@@ -28,6 +28,20 @@
 #include "platform/SingletonPtr.h"
 #include "platform/PlatformMutex.h"
 #include "platform/NonCopyable.h"
+#include <algorithm>
```

Comment:
may be a bit of an overkill for 'max' - consider using a small macro/inline instead

Reply:
Would agree if it was only used in the header file. However, as it is also used in the FlashIAP implementation, and assuming anyone that includes the header file would also use the FlashIAP class, then it shouldn't add any extra code.